File: docs/components_page/components/form/dash_core.jl
using DashBootstrapComponents

dropdown = html_div(
    [
        dbc_label("Dropdown", html_for = "dropdown"),
        dcc_dropdown(
            id = "dropdown",
            options = [
                Dict("label" => "Option 1", "value" => 1),
                Dict("label" => "Option 2", "value" => 2),
            ],
        ),
    ],
    className = "mb-3",
);

slider = html_div(
    [
        dbc_label("Slider", html_for = "slider"),
        dcc_slider(id = "slider", min = 0, max = 10, step = 0.5, value = 3),
    ],
    className = "mb-3",
);

range_slider = html_div(
    [
        dbc_label("RangeSlider", html_for = "range-slider"),
        dcc_rangeslider(id = "range-slider", min = 0, max = 10, value = [3, 7]),
    ],
    className = "mb-3",
);

form = dbc_form([dropdown, slider, range_slider]);

File: components_page/components/form/dash_core.py
import dash_bootstrap_components as dbc
from dash import dcc, html

dropdown = html.Div(
    [
        dbc.Label("Dropdown", html_for="dropdown"),
        dcc.Dropdown(
            id="dropdown",
            options=[
                {"label": "Option 1", "value": 1},
                {"label": "Option 2", "value": 2},
            ],
        ),
    ],
    className="mb-3",
)

slider = html.Div(
    [
        dbc.Label("Slider", html_for="slider"),
        dcc.Slider(id="slider", min=0, max=10, step=0.5, value=3),
    ],
    className="mb-3",
)

range_slider = html.Div(
    [
        dbc.Label("RangeSlider", html_for="range-slider"),
        dcc.RangeSlider(id="range-slider", min=0, max=10, value=[3, 7]),
    ],
    className="mb-3",
)

form = dbc.Form([dropdown, slider, range_slider])
